lib/matplotlib/figure.py

```diff
@@ -331,8 +331,9 @@ def _suplabels(self, t, info, **kwargs):
     @_docstring.copy(_suplabels)
     def suptitle(self, t, **kwargs):
         # docstring from _suplabels...
-        info = {'name': '_suptitle', 'x0': 0.5, 'y0': 0.98,
-                'ha': 'center', 'va': 'top', 'rotation': 0,
+        info = {'name': '_suptitle', 'x0': 'figure.title_x',
```

Comment:
I do not understand how `x0` and `y0` get turn from the rcParam names into actual values?

My guess is that this is related to the failures in the doc build